impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -32,13 +41,82 @@
  */
 public class Utils {
     private static final Logger log = getLogger(Utils.class);
+    private static final String CONSISTENCY_TEST = "__consistencyTest";
 
     private Utils() {
     }
 
     /**
      * Await expected broker count in cluster.
-     * Verifies that each broker in cluster is returning the expected cluster size.
+     * Verifies that each broker in the cluster is returning the expected cluster size.
+     *
+     * @param connectionConfig the connection config
+     * @param timeout the timeout
+     * @param timeUnit the time unit
+     * @param expectedBrokerCount the expected broker count
+     */
+    public static void awaitExpectedBrokerCountInClusterViaTopic(Map<String, Object> connectionConfig, int timeout, TimeUnit timeUnit, Integer expectedBrokerCount) {
+        try (var admin = Admin.create(connectionConfig)) {
```

Comment:
I think a couple of sentences explaining how we check that all brokers are present would be good.